data/mods/Aftershock/items/tools.json

```
@@ -24,6 +24,15 @@
     ],
     "flags": [ "WATCH", "ALARMCLOCK", "NO_UNLOAD", "NO_RELOAD" ]
   },
+  {
+    "type": "TOOL",
```

For dumb reasons, this needs to be of type `AMMO`, because it copy-from's `rock`.
```suggestion
    "type": "AMMO",
```